SCOPE.md

```diff
@@ -0,0 +1,53 @@
+# Scope
+
+The REST API specification is meant to provide the functionality currently offer by Wikibase "Action API".
+
+The scope of the schema is limited to the primary entity types (Items & Properties) and the main [CRU(D)](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) actions for those entities.
+
+## Intentionally not implemented
+
+The following elements are considered non essential for the design decisions and are intentionally excluded from the prototype schema:
+
+### Deletions
+
+As entity specific deletion is not yet covered by any Wikibase API the deletion part of CRUD is not covered in this specification.
+
+### Search
+
+A `wbsearchentities` equivalent has not been included in the specification.
+
+Investigations will be carried out to determine if such functionality may make sense to be included as part of the primary entity retrieval endpoint, or if search should remain a separate topic.
+
+### Badges
+
+A `wbavailablebadges` equivalent has not been included in the specification.
+
+The badges API distracts from the main focus and complexity of the proposed specification (getting and interacting with entities).
+
+It's likely that badges, or some way of retrieving lists of entities would be included as a separate endpoint.
+
+### UI focused modules
+
+The existing modules serve mainly as part of the Wikibase UI editing and display workflow and are not deemed to be used much externally.
+
+- `wbformatentities`
+- `wbformatvalue`
+- `wbparsevalue`
+
+### Client focused modules
+
+- `wbsearch`
```

Comment:
I'm not sure what this module is or how it differs from `wbsearchentities`. It is not documented as far as I can tell.

Reply:
In fact, I can't find docs for any of these modules

Reply:
If you ever can't find things in the API action docs, it is likely because it is not for a top level module.

https://www.wikidata.org/w/api.php?action=help&recursivesubmodules=1 loads all of the docs at once.

https://www.wikidata.org/w/api.php?action=help&recursivesubmodules=1#query+wbsearch

Reply:
Oh cool, thanks! I wasn't even aware that there were sub modules in our action API.